marlowe-website/src/index.njk

```diff
@@ -15,245 +15,414 @@
-      make it work that way. #}
-      <div class="grid grid-cols-1 md:grid-cols-2 widest:mx-auto max-w-screen h-full">
-        <div class="bg-market-banner md:bg-none bg-no-repeat bg-cover bg-pos-market-banner-mobile flex flex-col i items-start pl-4 md:pl-0 md:items-center pt-8 md:pt-0 md:justify-center">
-          <img loading="lazy" class="h-play-img-sm md:h-play-img-md" src="{% static './static/img/marlowe-market-coming-soon.svg' %}" alt="Marlowe play Update coming 2021 on the third or fourth quarter">
-        </div>
-        <div class="hidden md:block w-max">
-          <img loading="lazy" src="{% static './static/img/marlowe-market-banner-background-desktop.png' %}" alt="Woman using marlowe market on her cellphone">
+    <!--S1 - Marlowe Run-->
+
+    <section class="relative block bg-black h-auto w-full md:pt-24 md:pb-20 pt-16 pb-24 text-white">
+      <!--S1 - Marlowe Build-->
+
+      <div class="mx-3 md:mx-5vw flex justify-center">
+        {# Defines the margins #}
+
+        <div class="text-center text-white lg:mb-12 mb-8 z-10 w-full max-w-screen-widest">
+          <!--Row-->
+          <div class="">
+            <!--Column-->
+            <img loading="lazy" class="mx-auto lg:pb-0 md:pb-8 pb-8 lg:h-logo-wider h-logo" src="{% static './static/img/marlowe-build-demo-logo-dark.svg' %}" alt="">
+            <h3 class="md:text-3xl lg:text-4xl flex flex-col lg:font-comfortaa-bolder font-comfortaa-bold leading-tight lg:pt-8 pb-4 text-27px">Playground for financial engineers</h3>
+            <h2 class="text-xl lg:text-2xl xl:text-27px pb-0">Build in the language made for finance</h2>
+          </div>
+          <!--Column-->
         </div>
+        <!--Row-->
 
       </div>
-    </section>
-    <section class="pt-6 pb-20 lg:py-20 grid grid-cols-1 md:grid-cols-2 md:auto-rows-fr md:gap-x-6 lg:gap-x-5vw xl:gap-x-10vw mx-6 lg:mx-5vw xl:mx-10vw max-w-screen widest:mx-auto">
-      <div>
-        <h4 class="text-lightpurple text-xl font-semibold mb-6">The core of Marlowe is our special-purpose language.</h4>
-        <h3 class="text-2xl font-comfortaa-bold leading-tight mb-6">Marlowe is a new language for modeling financial contracts as smart contracts on a blockchain.</h3>
-        <p class="text-lg mb-6">Access the benefits of the Cardano blockchain to create financial contracts that are secure, automated, and easy to manage.</p>
-        <a href="" target="_blank" class="primary-button mb-6">Learn more</a>
-      </div>
-      <div class="md:flex gap-4">
-        <div class="py-6">
-          <h4 class="font-bold">Designed for finance</h4>
-          <p>Marlowe contracts are written in the language of finance.</p>
 
+      <div class="mx-3 md:mx-5vw flex justify-center">
+        {# Defines the margins #}
+
+        <div class="relative grid grid-cols-1 lg:grid-cols-2 w-full max-w-screen-widest z-10">
+          <!--Row-->
+
+          <div class="lg:w-10/12 w-11/12 self-center justify-self-center">
```

Comment:
are you using `w-10/12` and `11/12` for margins? if so please change to use proper margins.